src/OrbitGl/GraphTrack.cpp

```diff
@@ -259,15 +273,16 @@ template <size_t Dimension>
 void GraphTrack<Dimension>::DrawSingleSeriesEntry(
     Batcher* batcher, uint64_t start_tick, uint64_t end_tick,
     const std::array<float, Dimension>& normalized_cumulative_values, float z) {
-  float x0 = time_graph_->GetWorldFromTick(start_tick);
-  float width = time_graph_->GetWorldFromTick(end_tick) - x0;
-  float content_height = GetGraphContentHeight();
-  float base_y = GetGraphContentBaseY();
+  const float x0 = time_graph_->GetWorldFromTick(start_tick);
```

Comment:
I like those `const`s. Maybe we should do a pass (after this PR) to add consts where ever possible.